packages/dds/map/src/test/mocha/directory.spec.ts

```diff
@@ -249,8 +252,8 @@ describe("Directory", () => {
                 try {
                     subDirectory.set("throw", "error");
                     assert.fail("Should throw usage error");
-                } catch (error: any) {
-                    assert.strictEqual(error.errorType, "usageError", "Should throw usage error");
+                } catch (error) {
+                    assert.strictEqual((error as UsageError)?.errorType, "usageError", "Should throw usage error");
```

Comment:
Not quite sure I follow the error typing/casting here?  But no objections if you're confident this is reasonable.

Reply:
In this case the error is typed as `any`, which upsets the linter. Since the test is specifically looking for this kind of error, this should give us the check we want (if `errorType` is undefined, the test will fail).